sys/posix/include/netdb.h

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright (C) 2021 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup posix
+ *
+ * @{
+ * @file
+ * @brief  posix netdb implementation
+ * @author Hendrik van Essen <hendrik.ve@fu-berlin.de>
+ */
+
+#ifndef NETDB_H
+#define NETDB_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "sys/bytes.h"
+
+/** Errors used by the DNS API functions */
+/**
+ * @brief The name could not be resolved at this time. Future attempts may succeed.
+ */
+#define EAI_AGAIN       1
```

Comment:
```suggestion
#ifndef EAI_AGAIN 
#define EAI_AGAIN       1
#endif
```

would be the simplest solution